chains/bad_v2_delegate_stubbed.json

```diff
@@ -1,60 +1,60 @@
 {
-   "username" : "max32",
```

Comment:
I made a mistake when I first created these by not making sure that everything came out in alphabetical order, to keep these diffs from blowing up every time we make a small change. Would that be easy to fix now?